c/busybox-1.22.0/chroot-incomplete_false-no-overflow.i

```diff
@@ -301,26 +301,30 @@ typedef struct _IO_FILE FILE;
 extern void setpwent (void);
 extern void endpwent (void);
 extern struct passwd *getpwent (void);
-extern struct passwd *fgetpwent (FILE *__stream);
+extern struct passwd *fgetpwent (FILE *__stream) __attribute__ ((__nonnull__ (1)));
```

Comment:
This is a surprising change?

Reply:
This is the work of preprocessor (`gcc -E -P`). Different system, different headers. This is why the original commit was not using header files and manually just inserted the code where needed.

Reply:
Ok.